guide/src/commands/queries/packet.md

```diff
@@ -262,11 +262,11 @@ Success: [
 
 ## Unreceived Acknowledgments
 
-Use the `query packet unreceived-acks` command to query the sequence numbers of all packets that have not yet been acknowledged.
+Use the `query packet pending-acks` command to query the sequence numbers of all packets that have not yet been acknowledged.
 
 ```shell
 USAGE:
-    hermes query packet unreceived-acks --chain <CHAIN_ID> --port <PORT_ID> --channel <CHANNEL_ID>
+    hermes query packet pending-acks --chain <CHAIN_ID> --port <PORT_ID> --channel <CHANNEL_ID>
 
 DESCRIPTION:
     Query unreceived acknowledgments
```

Comment:
Should we replace all occurrences of `unreceived` with `pending`? (in code, comments and the guide?)

Reply:
Fixed the guide, let's do code changes after rc0.